### DISCUSSION TITLE
Fix Astral sun calculations

### DISCUSSION THREAD

**Discussion by jbouwh** [jbouwh/omnikdatalogger#81] [Announcements]
## What's Changed
* Release 1.11.3 by @jbouwh in https://github.com/jbouwh/omnikdatalogger/pull/80

Special Thanx to @MarijnS95 for the PR's to solve this.

**Full Changelog**: https://github.com/jbouwh/omnikdatalogger/compare/v1.11.1...v1.11.3

<hr /><em>This discussion was created from the release <a href='https://github.com/jbouwh/omnikdatalogger/releases/tag/v1.11.3'>Fix Astral sun calculations</a>.</em>

**Comment by MarijnS95** [comment]
Interesting release model to squash-merge `dev` into `main` :wink: (it makes it so that the release notes read: "Release 1.11.3" which isn't very intuitive :stuck_out_tongue:)

**Comment by jbouwh** [comment]
Just being lazy. Normally would prefer cherry picking. But I managed to get the release to Docker and PyPi. Also HACS will have a new release. Thnx for your contribution.

Marijn Suijten ***@***.***> schreef op 29 april 2022 19:53:13 CEST:
>Interesting release model to squash-merge `dev` into `main` :wink: 
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/jbouwh/omnikdatalogger/discussions/81#discussioncomment-2663190
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Reply by MarijnS95** [reply]
Ah, you can also tell GitHub to rebase-merge which is pretty much identical to manually cherry-picking patch-for-patch, but it's a bit finicky since it always ends up selecting the wrong merge method by default if you have >1 method enabled.
